flink-table/flink-sql-parser/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -177,6 +177,87 @@ SqlDescribeDatabase SqlDescribeDatabase() :
 
 }
 
+SqlCreate SqlCreateFunction(Span s, boolean replace) :
+{
+    SqlIdentifier functionName = null;
+    SqlCharStringLiteral functionClassName = null;
+    String functionLanguage = null;
```

Comment:
If we define language enum in sql parser module. It can't be accessed in api module. Thus, I would prefer to keep the value as String here, and add the enum in api module. We may do the transform in SqlToOperationConverter.